ui/component/subscribeButton/view.jsx

```diff
@@ -112,14 +122,27 @@ export default function SubscribeButton(props: Props) {
           onClick={() => {
             const newNotificationsDisabled = !notificationsDisabled;
 
-            doChannelSubscribe({
-              channelName: claimName,
-              uri: permanentUrl,
-              notificationsDisabled: newNotificationsDisabled,
-            }, false);
+            doChannelSubscribe(
```

Comment:
I'm not sure why some of this code was re-formatted differently. Need to look in to that.